templates/submission/internal-error-message.html

```diff
@@ -1,9 +1,10 @@
 <h3 style="color:red;font-weight:bold">
     {% if request.user == submission.user.user %}
         {% trans trimmed %}
```

Comment:
Is the `trimmed` necessary? If not, could we switch this to something similar to my above comment?

Reply:
I changed `trans` to `_()`.

`trimmed` is needed so that Crowdin's translation string doesn't have whitespace everywhere.